README.md

```diff
@@ -0,0 +1,38 @@
+# Roulette Telegram Chat
+
+![Man speaking with women](./docs/logo.png)
+The idea is to create a chat for communication between two random people without revealing their identities.
+
+## Use case diagram
+
+![Interaction between user, moderator and system](./docs/useCase.png)
+Main points:
+
+### User
+
+- can register in system(by pressing button start)
+- can change companion to new one
+- can stop communication
```

Comment:
how you will separate communications in chat from commands?

Reply:
the commands will start with `/` sign, the user commonly don't use `/` in their conversation

Reply:
ok